keyboards/vortex/m0110/info.json

```diff
@@ -0,0 +1,402 @@
+                { "label": "[", "matrix": [8, 1], "x": 11.5, "y": 2 },
+                { "label": "]", "matrix": [6, 1], "x": 12.5, "y": 2 },
+                { "label": "\\", "matrix": [10, 5], "w": 1.5, "x": 13.5, "y": 2 },
+
+                { "label": "Caps Lock", "matrix": [2, 1], "w": 1.75, "x": 0, "y": 3 },
+                { "label": "A", "matrix": [1, 2], "x": 1.75, "y": 3 },
+                { "label": "S", "matrix": [2, 2], "x": 2.75, "y": 3 },
+                { "label": "D", "matrix": [3, 2], "x": 3.75, "y": 3 },
+                { "label": "F", "matrix": [4, 2], "x": 4.75, "y": 3 },
+                { "label": "G", "matrix": [4, 3], "x": 5.75, "y": 3 },
+                { "label": "H", "matrix": [5, 3], "x": 6.75, "y": 3 },
+                { "label": "J", "matrix": [5, 2], "x": 7.75, "y": 3 },
+                { "label": "K", "matrix": [6, 2], "x": 8.75, "y": 3 },
+                { "label": "L", "matrix": [7, 2], "x": 9.75, "y": 3 },
+                { "label": ";", "matrix": [8, 2], "x": 10.75, "y": 3 },
+                { "label": "'", "matrix": [8, 3], "x": 11.75, "y": 3 },
+                { "label": "#", "matrix": [0, 2], "x": 12.75, "y": 3 },
+                { "label": "Enter", "matrix": [10, 4], "w": 1.25, "x": 13.75, "y": 3 },
+
+                { "label": "Shift", "matrix": [0, 0], "w": 1.25, "x": 0, "y": 4 },
+                { "label": "\\", "matrix": [2, 3], "x": 1.25, "y": 4 },
+                { "label": "Z", "matrix": [1, 4], "x": 2.25, "y": 4 },
+                { "label": "X", "matrix": [2, 4], "x": 3.25, "y": 4 },
+                { "label": "C", "matrix": [3, 4], "x": 4.25, "y": 4 },
+                { "label": "V", "matrix": [4, 4], "x": 5.25, "y": 4 },
+                { "label": "B", "matrix": [4, 5], "x": 6.25, "y": 4 },
+                { "label": "N", "matrix": [5, 5], "x": 7.25, "y": 4 },
+                { "label": "M", "matrix": [5, 4], "x": 8.25, "y": 4 },
+                { "label": ",", "matrix": [6, 4], "x": 9.25, "y": 4 },
+                { "label": ".", "matrix": [7, 4], "x": 10.25, "y": 4 },
+                { "label": "/", "matrix": [8, 5], "x": 11.25, "y": 4 },
+                { "label": "Shift", "matrix": [9, 1], "w": 2.75, "x": 12.25, "y": 4 },
+                                                
+                { "label": "Ctrl", "matrix": [0, 6], "w": 1.25, "x": 0, "y": 5 },
+                { "label": "Win", "matrix": [9, 0], "w": 1.25, "x": 1.25, "y": 5 },
+                { "label": "Alt", "matrix": [9, 3], "w": 1.25, "x": 2.5, "y": 5 },
+                { "label": "Space", "matrix": [9, 4], "w": 6.25, "x": 3.75, "y": 5 },
+                { "label": "RAlt", "matrix": [9, 5], "w": 1.25, "x": 10, "y": 5 },
+                { "label": "RWin", "matrix": [8, 4], "w": 1.25, "x": 11.25, "y": 5 },
+                { "label": "RCtrl", "matrix": [0, 4], "w": 1.25, "x": 12.5, "y": 5 },
+                { "label": "FN", "matrix": [9, 2], "w": 1.25, "x": 13.75, "y": 5 }
```

Comment:
Moves the 2u/Split Backspace keys around to visually clarify which are which:

![image](https://github.com/qmk/qmk_firmware/assets/18669334/82b6c83c-3c70-464b-8b80-f8a6e5542e08)

:warning: Requires companion changes to both `keymap.c` files.

```suggestion
                {"label": "ESC", "matrix": [1, 6], "x": 0, "y": 0},
                {"label": "1", "matrix": [1, 7], "x": 1, "y": 0},
                {"label": "2", "matrix": [2, 7], "x": 2, "y": 0},
                {"label": "3", "matrix": [3, 7], "x": 3, "y": 0},
                {"label": "4", "matrix": [4, 7], "x": 4, "y": 0},
                {"label": "5", "matrix": [4, 6], "x": 5, "y": 0},
                {"label": "6", "matrix": [5, 6], "x": 6, "y": 0},
                {"label": "7", "matrix": [5, 7], "x": 7, "y": 0},
                {"label": "8", "matrix": [6, 7], "x": 8, "y": 0},
                {"label": "9", "matrix": [7, 7], "x": 9, "y": 0},
                {"label": "0", "matrix": [8, 7], "x": 10, "y": 0},
                {"label": "-", "matrix": [8, 6], "x": 11, "y": 0},
                {"label": "=", "matrix": [6, 6], "x": 12, "y": 0},
                {"label": "\\", "matrix": [10, 2], "x": 13, "y": 0.5, "h": 0.5},
                {"label": "Backspace", "matrix": [10, 1], "x": 13, "y": 0, "w": 2, "h": 0.5},
                {"label": "del", "matrix": [6, 5], "x": 14, "y": 0.5, "h": 0.5},

                {"label": "Tab", "matrix": [1, 1], "w": 1.5, "x": 0, "y": 1},
                {"label": "Q", "matrix": [1, 0], "x": 1.5, "y": 1},
                {"label": "W", "matrix": [2, 0], "x": 2.5, "y": 1},
                {"label": "E", "matrix": [3, 0], "x": 3.5, "y": 1},
                {"label": "R", "matrix": [4, 0], "x": 4.5, "y": 1},
                {"label": "T", "matrix": [4, 1], "x": 5.5, "y": 1},
                {"label": "Y", "matrix": [5, 1], "x": 6.5, "y": 1},
                {"label": "U", "matrix": [5, 0], "x": 7.5, "y": 1},
                {"label": "I", "matrix": [6, 0], "x": 8.5, "y": 1},
                {"label": "O", "matrix": [7, 0], "x": 9.5, "y": 1},
                {"label": "P", "matrix": [8, 0], "x": 10.5, "y": 1},
                {"label": "[", "matrix": [8, 1], "x": 11.5, "y": 1},
                {"label": "]", "matrix": [6, 1], "x": 12.5, "y": 1},
                {"label": "\\", "matrix": [10, 5], "w": 1.5, "x": 13.5, "y": 1},

                {"label": "Caps Lock", "matrix": [2, 1], "w": 1.75, "x": 0, "y": 2},
                {"label": "A", "matrix": [1, 2], "x": 1.75, "y": 2},
                {"label": "S", "matrix": [2, 2], "x": 2.75, "y": 2},
                {"label": "D", "matrix": [3, 2], "x": 3.75, "y": 2},
                {"label": "F", "matrix": [4, 2], "x": 4.75, "y": 2},
                {"label": "G", "matrix": [4, 3], "x": 5.75, "y": 2},
                {"label": "H", "matrix": [5, 3], "x": 6.75, "y": 2},
                {"label": "J", "matrix": [5, 2], "x": 7.75, "y": 2},
                {"label": "K", "matrix": [6, 2], "x": 8.75, "y": 2},
                {"label": "L", "matrix": [7, 2], "x": 9.75, "y": 2},
                {"label": ";", "matrix": [8, 2], "x": 10.75, "y": 2},
                {"label": "'", "matrix": [8, 3], "x": 11.75, "y": 2},
                {"label": "#", "matrix": [0, 2], "x": 12.75, "y": 2},
                {"label": "Enter", "matrix": [10, 4], "w": 1.25, "x": 13.75, "y": 2},

                {"label": "Shift", "matrix": [0, 0], "w": 1.25, "x": 0, "y": 3},
                {"label": "\\", "matrix": [2, 3], "x": 1.25, "y": 3},
                {"label": "Z", "matrix": [1, 4], "x": 2.25, "y": 3},
                {"label": "X", "matrix": [2, 4], "x": 3.25, "y": 3},
                {"label": "C", "matrix": [3, 4], "x": 4.25, "y": 3},
                {"label": "V", "matrix": [4, 4], "x": 5.25, "y": 3},
                {"label": "B", "matrix": [4, 5], "x": 6.25, "y": 3},
                {"label": "N", "matrix": [5, 5], "x": 7.25, "y": 3},
                {"label": "M", "matrix": [5, 4], "x": 8.25, "y": 3},
                {"label": ",", "matrix": [6, 4], "x": 9.25, "y": 3},
                {"label": ".", "matrix": [7, 4], "x": 10.25, "y": 3},
                {"label": "/", "matrix": [8, 5], "x": 11.25, "y": 3},
                {"label": "Shift", "matrix": [9, 1], "w": 2.75, "x": 12.25, "y": 3},

                {"label": "Ctrl", "matrix": [0, 6], "w": 1.25, "x": 0, "y": 4},
                {"label": "Win", "matrix": [9, 0], "w": 1.25, "x": 1.25, "y": 4},
                {"label": "Alt", "matrix": [9, 3], "w": 1.25, "x": 2.5, "y": 4},
                {"label": "Space", "matrix": [9, 4], "w": 6.25, "x": 3.75, "y": 4},
                {"label": "RAlt", "matrix": [9, 5], "w": 1.25, "x": 10, "y": 4},
                {"label": "RWin", "matrix": [8, 4], "w": 1.25, "x": 11.25, "y": 4},
                {"label": "RCtrl", "matrix": [0, 4], "w": 1.25, "x": 12.5, "y": 4},
                {"label": "FN", "matrix": [9, 2], "w": 1.25, "x": 13.75, "y": 4}
```